### ISSUE TITLE
Error during the build on version 17.12.0

### ISSUE THREAD

**Issue by DianaSolanoR** [primefaces/primeng-sass-theme#117]
Hello,

I just upgraded the library version to 17.12.0. I'm using the lara-cyan theme and I encountered the following error during the build:

**Undefined variable.
21 │     outline-offset: $inputFocusOutlineOffset;
node_modules/primeng-sass-theme/theme-base/_mixins.scss 21:21                         focused-input()
node_modules/primeng-sass-theme/theme-base/components/input/_autocomplete.scss 22:13  
@import node_modules/primeng-sass-theme/theme-base/_components.scss 10:13                     
@import node_modules/primeng-sass-theme/themes/lara/lara-dark/cyan/theme.scss 13:9** 

It appears that the variable does not exist in the _variables.scss file of that theme. Could you please take a look?

Thank you very much and best regards.

**Comment by SoyDiego** [comment]
Hi, I have checked the file and exists the variable in the [line 58](https://github.com/primefaces/primeng-sass-theme/blob/main/themes/lara/lara-light/_variables.scss#L58)

![image](https://github.com/primefaces/primeng-sass-theme/assets/19764334/1c7d5aeb-ba92-47cc-ade0-d02c173ee180)

Did you upgrade your library?

**Comment by DianaSolanoR** [comment]
Hello,
The issue exists in the dark version of the theme, not in the light one.

Thank you

**Comment by SoyDiego** [comment]
> Hello, The issue exists in the dark version of the theme, not in the light one.
> 
> Thank you

I have checked the dark and the line is included too:

![image](https://github.com/primefaces/primeng-sass-theme/assets/19764334/69919f8b-4e3d-401f-aa40-cb1af504e0b2)

Maybe I'm checking in the wrong place. Could you confirm @cetincakiroglu if this problem exists or if i'm in the correct file. If not tell me what is the file and I can make the PR

**Comment by christophercr** [comment]
Same issue here.

@SoyDiego it seems that what gets installed via NPM is not the same as the source files:

![image](https://github.com/primefaces/primeng-sass-theme/assets/5698318/3d343820-b5c8-4224-ab72-338896d0a84b)

Perhaps an issue in your deployment process?

**Comment by SoyDiego** [comment]
> Same issue here.
> 
> @SoyDiego it seems that what gets installed via NPM is not the same as the source files (both themes, version `17.12.0`):
> 
> ![image](https://private-user-images.githubusercontent.com/5698318/318742947-3d343820-b5c8-4224-ab72-338896d0a84b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNTkzODMsIm5iZiI6MTcxMjA1OTA4MywicGF0aCI6Ii81Njk4MzE4LzMxODc0Mjk0Ny0zZDM0MzgyMC1iNWM4LTQyMjQtYWI3Mi0zMzg4OTZkMGE4NGIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MDJUMTE1ODAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzAzNTYxMTczMWFiNGI1NGY4YmRkNTFjZWI5MjYwYjUzMjE2NjVkYWZkM2Q2ODk1YTM1MmE4ZTZhNjQ5OTRhYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.aFNsmYWTG4-7_iIu0LLo4gMJywW9qOlLZTpTx3fv7_M)
> 
> Perhaps an issue in your publish process?

Are you using NPM to install it? 
I think Primetek doesn't have a official package for that. Can you share the link?
Always I cloned the project and compiled my styles and then my final `.css` files I used in my projects.

**Comment by christophercr** [comment]
Indeed, I'm using NPM 10. And I thought I could simply install the `primeng-sass-theme` package to avoid doing that manually every time there is a new release:

`"primeng-sass-theme": "github:primefaces/primeng-sass-theme#17.11.0",`

But as I can see, there are some differences between the source code and the code that gets "installed" using NPM.

As I said, this was my initial idea to reduce the manual work in maintaining the theme in sync with PrimeNG. In fact, it seems I'm not the only one who wants to avoid this manual work: in https://github.com/primefaces/primeng-sass-theme/issues/62

Do you think this is something the PrimeNG team can have a look at? At least ensuring that installing via `github:primefaces/primeng-sass-theme#17.11.0` gets the same code as if we would download the ZIP file from https://github.com/primefaces/primeng-sass-theme/releases

**Comment by christophercr** [comment]
Maybe this PR makes it work? https://github.com/primefaces/primeng-sass-theme/pull/113

**Comment by christophercr** [comment]
Digging a bit more into this, I think there is a mismatch in the tag `17.12.0` itself. I mean, checking the source code directly in that tag (https://github.com/primefaces/primeng-sass-theme/blob/17.12.0/themes/lara/lara-light/_variables.scss) , I can see that the variable `$inputFocusOutlineOffset;` is indeed missing:

![image](https://github.com/primefaces/primeng-sass-theme/assets/5698318/8ade12e3-facc-448f-b641-9ef33e9d6b4a)

**Comment by SoyDiego** [comment]
> Indeed, I'm using NPM 10. And I thought I could simply install the `primeng-sass-theme` package to avoid doing that manually every time there is a new release:
> 
> `"primeng-sass-theme": "github:primefaces/primeng-sass-theme#17.12.0",`
> 
> But as I can see, there are some differences between the source code and the code that gets "installed" using NPM.
> 
> As I said, this was my initial idea to reduce the manual work in maintaining the theme in sync with PrimeNG. In fact, it seems I'm not the only one who wants to avoid this manual work: #62
> 
> Do you think this is something the PrimeNG team can have a look at? At least ensuring that installing via `github:primefaces/primeng-sass-theme#17.12.0` gets the same code as if we would manually download the ZIP file from https://github.com/primefaces/primeng-sass-theme/releases

I'm not part of Primetek only a contributor but I think they are not interested (I don't remember why in create a NPM package, **maybe i'm wrong, I don't know 100%**).

**Comment by SoyDiego** [comment]
> Digging a bit more into this, I think there is a mismatch in the tag `17.12.0` itself. I mean, checking the source code directly in that tag (https://github.com/primefaces/primeng-sass-theme/blob/17.12.0/themes/lara/lara-light/_variables.scss) , I can see that the variable `$inputFocusOutlineOffset;` is indeed missing:
> 
> ![image](https://private-user-images.githubusercontent.com/5698318/318802881-8ade12e3-facc-448f-b641-9ef33e9d6b4a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNjU4NDcsIm5iZiI6MTcxMjA2NTU0NywicGF0aCI6Ii81Njk4MzE4LzMxODgwMjg4MS04YWRlMTJlMy1mYWNjLTQ0OGYtYjY0MS05ZWYzM2U5ZDZiNGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MDJUMTM0NTQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWJjMGQ3NjZhOGZmMzdkOWEzOWJkYmI5YmFmYmRkMjNjNmY4ODIxMzNiNzVkMGUyNjY4ZjEzMWZkYThlN2MzYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.rd_q4p-Bg6mOYnNjnnRWSq9mMYQYzOC7MiUDloIXDPo)
> 
> And there is a newer commit in the `main` branch adding such missing variable: [07ee7cf](https://github.com/primefaces/primeng-sass-theme/commit/07ee7cfd98df828a9bdd31eb4f5424e700832409)
> 
> So I guess it's just matter or replacing the latest `17.12.0` release to point to the latest commit

If I understood ok, maybe they created the tag before this commit. Will see if generating a new tag, the problem is solved. We need to wait for Primetek team

**Comment by christophercr** [comment]
Yeah, that's right. Fortunately in the meantime I can simply add the commit hash to get the latest changes:

`"primeng-sass-theme": "primefaces/primeng-sass-theme#07ee7cfd98df828a9bdd31eb4f5424e700832409",

**Comment by SoyDiego** [comment]
> Yeah, that's right. Fortunately in the meantime I can simply add the commit hash to get the latest changes:
> 
> `"primeng-sass-theme": "primefaces/primeng-sass-theme#07ee7cfd98df828a9bdd31eb4f5424e700832409"`

That's good alternative idea at the moment. Thanks for share it
